### ISSUE TITLE
Content on species pages is not always relevant

### ISSUE THREAD

**Issue by sophiathirza** [nbnuk/nbnatlas-issues#388]
The content we pull in from EoL is not always UK centred  (#381)  and sometimes is not in english (#313). There are also problems with incorrect images being pulled in from EoL (#349 ).

The options are:
* find another data source other than EoL
* create a black list mechanism for EoL pages that are problematic
* correct, or create appropriate, EoL pages

**Comment by sophiathirza** [comment]
I have not found an alternative data source. Creating a list of problematic pages whose content are not displayed is the quickest solution. 

We should be reporting incorrect images and content to EoL.

**Comment by MNSmith** [comment]
Why not speak to the various National Schemes and Societies?  Many of these have websites with species accounts that could be linked to - I thought this was the idea for the start.  You will not find a single data source that will allow you to pull everything from one site, but you will find a lot of the resources you need are already present.  Abandon all use of EoL.

**Comment by sophiathirza** [comment]
Thank you for your comment. I agree that we should investigate this further. I suspect that staying with EoL may give us more work than looking for, and implementing, alternative data sources.

**Comment by sophiathirza** [comment]
In the short term, can we have a message added above the EoL information that makes it clear that it comes from EoL, please? I had another email today about it.

**Comment by sophiathirza** [comment]
#400

**Comment by sophiathirza** [comment]
Jo suggested looking at Wikipedia as a source of information on the species pages. Map of Life (https://mol.org) uses Wikipedia. I did a quick check of some of our problematic species and Wikipedia does look better than EoL; the descriptions are much more general.

**Comment by MNSmith** [comment]
I think this still misses the point that a "general, single species information source", while easy to set up a link for, will not have the level of accuracy in species information that the NBN Atlas should be offering when people are looking at a species info page.  Species pages should be individually linked to the best source of data that is around, hence my comments about the NSS pages.

**Comment by djtfmartin** [comment]
Using the content from NSS pages would be the ideal, but it will take a considerable amount of time to achieve technically (theres no widely adopted exchange standard for species profile information content) and there would need to be data agreements put in place. In additional, the national schemes may not wish to share this information.

**Comment by MNSmith** [comment]
I agree with your comment about the NSS not wishing to share info, but has anybody asked.

The NBN species page does not have to display a block of text  / import information from another page, a simple hyperlink to a reliable page of information would be sufficient.  What appears from EoL is not in a standard format at present.

**Comment by EllaVogel** [comment]
I agree that ideally, we'd have these sections editable by experts in NSS.

In the meantime, has anyone explored if arkive.org is a better source of info than EoL? E.g. http://www.arkive.org/puffin/fratercula-arctica/ . I've had a quick look and the info seems good, but I'm not an expert so would struggle to pick up too many species specific errors.

**Comment by djtfmartin** [comment]
The NBN Atlas species pages have the content from Arkive, and its coming from the EOL webservices.

<img width="598" alt="screenshot 2018-01-17 12 48 48" src="https://user-images.githubusercontent.com/444897/35043531-e18a20f0-fb84-11e7-870d-92313cea85fc.png">

If there are additional sources (e.g. NSS) that should be on NBN atlas species pages, the easiest thing to do might be to expose that to EOL (using their data exchange formats & licencing recommendations).

Then NBN could retrieve this content from the EOL without the need to build additional infrastructure to store, index and serve it via webservices.

**Comment by EllaVogel** [comment]
Someone recently suggested that Recorder 3 may be a good place to look for accurate species accounts. These have been written by species experts around the UK, although I'm not sure how accessible this information is. It may be worth getting in touch with someone at JNCC about access to these descriptions? http://jncc.defra.gov.uk/page-5008

**Comment by EllaVogel** [comment]
Sample species descriptions from Recorder 3 have been emailed through from John Bratton. I've have passed these onto Sophie via email, so can be followed up if they appear to be a good alternative.
